BangazonAPI/BangazonAPI/Models/Product.cs

```diff
@@ -5,7 +5,22 @@
 
 namespace BangazonAPI.Models
 {
-    public class Class
+    public class Product
     {
+        public int id { get; set; }
+        public int price { get; set; }
+        public string title { get; set; }
+        public string description { get; set; }
+        public int quantity { get; set; }
+        public List<ProductType> productType = new List<ProductType>();
```

Comment:
Based on the ERD you are missing the ProductType Id and CustomerId.
Remove the methods and list.
Is there anything else you should add here?